GameOfLife/checkbuttonpressed.cpp

```diff
@@ -1,23 +1,50 @@
 #include "checkbuttonpressed.h"
 
+bool CheckButtonPressed::getSaveToFileStatus()
+{
+    return saveToFileStatus;
+}
+
```

Comment:
Just after creating PR it was only reaname. The work was still going on, and unfortunatelly next commits were added to this PR. We need solution on this case like creating additional remote branch after PR, so new chenges will not be added to previous PR. Sorry my fault, I should have predicted this, next commits will be pushed to another branch.

Greetings!